tests/test_discrete_quadratic_model.py

```diff
@@ -13,12 +13,44 @@
 #    limitations under the License.
 
 import itertools
+import numbers
+import shutil
+import tempfile
 import unittest
 
 import dimod
 import numpy as np
 
 
+# will want to migrate this to generators at some point
+def gnp_random_dqm(num_variables, num_cases, p, p_case, seed=None):
+    np.random.seed(seed)
```

Comment:
To limit function side-effects, you can use [`numpy.random.default_rng`](https://numpy.org/doc/stable/reference/random/generator.html#numpy.random.default_rng).

Reply:
Totally, I was in a hurry and this one lives only in testing. But good point, might as well to it correctly :smile: